<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="chatglm_textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatglm_lineEdit">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>380</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chatglm_pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
